bin/summariser.py

```diff
@@ -101,6 +105,40 @@ def runprocess(db_config_file, config_file, log_config_file):
         sys.exit(1)
 
     log.info('Created Pidfile')
+
+    # Configure options for stale summary clean up.
+    try:
+        stale_summary_clean_up = cp.getboolean('stale_summaries',
+                                               'enable_clean_up')
+
+    except ConfigParser.NoSectionError as error:
+        # If section no defined, assume the user has made no effort to
+        # configure stale summary clean up, in which case - just log the
+        # exception as info.
+        log.info("Will not clean up stale summarises.")
+        stale_summary_clean_up = False
+
+    except ConfigParser.NoOptionError as error:
+        # If the section is defined and the option is not, assume the user has
+        # made effort to configure stale summary clean up, in which case - log
+        # the exception it as a warning.
+        log.warn("Will not clean up stale summarises.")
```

Comment:
These and the other two in this file should be `.warning()`  now. `.warn()` is deprecating.